docs/consuming_with_cmake.rst

```diff
@@ -0,0 +1,91 @@
+.. _consuming_with_cmake:
+
+Consuming with CMake
+==================== 
+
+The best way to use oneMKL is through integration with CMake. This can be done in two ways:
+
+* By using ``find_package`` with an installed version of oneMKL interface library.
+* By downloading and building oneMKL as part of your application using FetchContent_.
```

Comment:
What is "FetchContent"? Link?

Reply:
This links to the `FetchContent` page in the CMake documentation. The link itself is defined at the bottom of the `consuming_with_cmake.rst` file.

Reply:
I did not know about this way of making a markdown link!